lib/eslint/eslint-helpers.js

```diff
@@ -13,9 +13,17 @@ const path = require("path");
 const fs = require("fs");
 const fsp = fs.promises;
 const isGlob = require("is-glob");
-const globby = require("globby");
 const hash = require("../cli-engine/hash");
 const minimatch = require("minimatch");
+const util = require("util");
+const fswalk = require("@nodelib/fs.walk");
+const { Minimatch } = require("minimatch");
```

Comment:
there's two requires of `minimatch`, should they perhaps all or none be destructured?

Reply:
Hmmm, I suspect this was VS Code being "helpful".